content/reference-rippled.md

```diff
@@ -737,6 +741,181 @@ The response follows the [standard format](#response-formatting), with a success
+
+#### Response Format
+
+An example of a successful response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+  "id": 2,
+  "status": "success",
+  "type": "response",
+  "result": {
+    "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+    "channels": [
+      {
+        "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+        "amount": "100000000",
+        "balance": "1000000",
+        "channel_id": "5DB01B7FFED6B67E6B0414DED11E051D2EE2B7619CE0EAA6286D67A3A4D5BDB3",
+        "destination_account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+        "destination_tag": 20170428,
+        "expiration": 547073182,
+        "public_key": "aB44YfzW24VDEJQ2UuLPV2PvqcPCSoLnL7y5M1EzhdW4LnK5xMS3",
+        "public_key_hex": "023693F15967AE357D0327974AD46FE3C127113B1110D6044FD41E723689F81CC6",
+        "settle_delay": 86400
+      }
+    ]
+  }
+}
+```
+
+*JSON-RPC*
+
+```json
+200 OK
+
+{
+    "result": {
+        "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+        "channels": [{
+            "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+            "amount": "100000000",
+            "balance": "0",
+            "channel_id": "5DB01B7FFED6B67E6B0414DED11E051D2EE2B7619CE0EAA6286D67A3A4D5BDB3",
+            "destination_account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            "destination_tag": 20170428,
+            "public_key": "aB44YfzW24VDEJQ2UuLPV2PvqcPCSoLnL7y5M1EzhdW4LnK5xMS3",
+            "public_key_hex": "023693F15967AE357D0327974AD46FE3C127113B1110D6044FD41E723689F81CC6",
+            "settle_delay": 86400
+        }],
+        "status": "success"
+    }
+}
+```
+
+*Commandline*
+
+```json
+200 OK
+
+{
+    "result": {
+        "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+        "channels": [{
+            "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+            "amount": "100000000",
+            "balance": "0",
+            "channel_id": "5DB01B7FFED6B67E6B0414DED11E051D2EE2B7619CE0EAA6286D67A3A4D5BDB3",
+            "destination_account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            "destination_tag": 20170428,
+            "public_key": "aB44YfzW24VDEJQ2UuLPV2PvqcPCSoLnL7y5M1EzhdW4LnK5xMS3",
+            "public_key_hex": "023693F15967AE357D0327974AD46FE3C127113B1110D6044FD41E723689F81CC6",
+            "settle_delay": 86400
+        }],
+        "status": "success"
+    }
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+The response follows the [standard format](#response-formatting), with a successful result containing the following fields:
+
+| Field      | Type                                       | Description        |
+|:-----------|:-------------------------------------------|:-------------------|
+| `account`  | String                                     | The address of the account from the request; the owner of the payment channels. |
```

Comment:
This is a little tricky, for a few reasons:

- Anyone can initiate a request for any account. The API request isn't "from" an account.
- It really means the `account` address from the request. However, the `account` value in the request may not actually be an address; it could be a public key or one or two other formats. Regardless of what format was used in the request, the response always formats `account` as an address. Thus, "the address of the account from the request".
- The "owner" and "source" refer to the same thing in payment channels. ("Source" emphasizes the direction the XRP is flowing and who created the channel and is a dualism to the term "destination" used in the API. "Owner" emphasizes who's responsible for the Owner Reserve, and who authorizes the flow of the XRP.)